ansible/roles/ocp4-workload-cnvlab/tasks/workload.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: Setting up workload for user
+  debug:
+    msg: "Setting up workload for user ocp_username = {{ ocp_username }}"
+
+# To Do: Implement your workload deployment tasks here
+# -------------------------------------------------------------------------
+
+- name: Create project for workbook
+  k8s:
+    state: present
+    definition: "{{ lookup('file', 'project.yaml') }}"
+
+- name: Clone OpenShift virt repositories
+  git:
+    repo: "https://github.com/RHFieldProductManagement/openshift-virt-labs.git"
+    dest: "~/openshift-virt-labs"
+    version: "rhpds"
+
+- name: Deploy Workbook Instance for CNV Labs
+  block:
+  - name: Set project to workbook
```

Comment:
Rather than setting the project it's much preferred to just use `-n workbook` on all subsequent commands (if they can't be replaced by `k8s`)

Reply:
Good catch